step.js

```diff
@@ -145,18 +164,24 @@ function moveShipDown() {
 
 function moveShipLeft() {
   if (airfighter.x > 0) {
-    airfighter.x -= 3;
+    airfighter.x += airfighter.vx;
+    if (airfighter.vx > -5) {
+      airfighter.vx -= acceleration / 2;
```

Comment:
Please create function for this action:
```
function decelerateAirfighter() {
  airfighter.vx -= acceleration / 2;
}
```

And create variable for ` if (airfighter.vx > -5)`:

```
const something = airfighter.vx > -5;
if (something) {
  decelerateAirfighter();
}
```

This way the code will be easier to read.